deploy/backplane/cssre/redhat-kas-fleetshard/10-mkrts-admins-project.ClusterRole.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-mkrts-users-project
```

Comment:
This will conflict with the https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/mkrts/redhat-kas-fleetshard/10-mkrts-admins-project.ClusterRole.yml

Reply:
What should this be?

Reply:
changed mkrts to cssre, I think that resolves this.